### ISSUE TITLE
Postgresql Import / Export is extremely slow and flawed

### ISSUE THREAD

**Issue by BFJonk** [TablePlus/TablePlus#3073]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
postgresql 14.7

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
Version 5.4.0 (505)

3. The steps to reproduce this issue:
I have a large table with 2.2 Million rows 
![image](https://github.com/TablePlus/TablePlus/assets/24241551/6e4d8551-0a62-49b5-9d12-8aca266ba577)

a. I Exported the table as CSV using tableplus. This took looooooooooooooooooooooong (2 **hours** or so?) and resulted in a file of 13,57 GB 
![image](https://github.com/TablePlus/TablePlus/assets/24241551/f84cee6e-41c1-4e46-8414-f740776347ae)

b. I tried to import the table using tableplus, that went even slower. In the end I cancelled the process (I have to admit that I used this process (with TablePlus) for large tables earlier, and I think it does work if you wait long enough ...... )

c. I decided to use pgAdmin. First, I tried to import the CSV generated by tableplus. It failed after 626 seconds with the following error: 
`ERROR: invalid input syntax for type date: "date"
CONTEXT: COPY fact_revenue, line 1, column date: "date"`
  
d. I decided to use pgAdmin for the export as well . The results were astonishing: 
Export : 17.95 seconds ;  Import 75.75 seconds   
![image](https://github.com/TablePlus/TablePlus/assets/24241551/4b89d2b0-eefd-4981-9b9c-c69270837c49)
File size CSV : 13.57 MB:
![image](https://github.com/TablePlus/TablePlus/assets/24241551/f84aaf7f-51ff-4751-8c06-dcb04b4268e4)

The file size and the failed import suggest that the export is flawed as well, next to being slow. 

Please let me know if you have further questions.

**Comment by rtconner** [comment]
Yeah I've never had a clean export/import in postgres with TablePlus. There is always some weird issue every time I do it and end up just having to do it command line with psql.

One issue I hit is TablePlus export is treating UUID's primary keys as integers and so of course that fails on import.
